### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-05-02

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#152] [General]
Previous agenda: #150

### Agenda
- [ ] Introduce any new attendees.
- [ ] Summarize any PRs merged since last meeting.
- [ ] Review release follow-up issues and PRs
- [ ] Nominate work for next release
- [x] Open next meeting discussion

**Comment by astroesteban** [comment]
Is there a newsletter or a post on the ROS Discourse with upcoming or scheduled meetings?

**Reply by EzraBrooks** [reply]
Hi @astroesteban, we have generally favored toward posting on GitHub rather than Discourse, as Space ROS is aiming to bridge the gap between organizations not using ROS and organizations that are - so GitHub is more of a common meeting ground.

I would be open to considering posting in both places, but we're not a huge project right now, so that's just one more bit of overhead we'd have to remember, so I'm not sure we'll be doing that yet.

**Comment by Bckempa** [comment]
Is anyone trying to join the meeting? Without @EzraBrooks we don't have permissions to add people to the room

**Comment by yuyuqq** [comment]
I tryed to enter.

2024年5月2日(木) 23:41 Bckempa ***@***.***>:

**Reply by Bckempa** [reply]
Sorry about that, Ezra is out of the office but he is the meeting host and that lead to some issues that we'll get ironed out so they don't happen again. I'll update this page with some meeting notes before next time but after we got a few people in a meeting together all we really had time for was discussing https://github.com/space-ros/docker/pull/150 which automates tagging of release images in docker hub

**Reply by EzraBrooks** [reply]
I've fixed the meeting permissions going forward, sorry about that!

**Comment by Bckempa** [comment]
Next meeting opened https://github.com/space-ros/space-ros/discussions/165
